packages/uploadthing/src/internal/handler.ts

```diff
@@ -151,6 +152,7 @@ export const buildRequestHandler = <TRouter extends FileRouter>(
     const uploadthingHook = getHeader(req, "uploadthing-hook") ?? undefined;
     const slug = params.get("slug") ?? undefined;
     const actionType = (params.get("actionType") as ActionType) ?? undefined;
+    const utPackage = getHeader(req, "x-ut-package") ?? "unknown";
```

Comment:
why is this `x-ut-package` when its `x-uploadthing-package` above?

Reply:
because I am not good at writing code.